nsxt/policy_common.go

```diff
@@ -469,6 +470,11 @@ func getPolicyRulesFromSchema(d *schema.ResourceData) []model.Rule {
 		}
 
 		resourceType := "Rule"
+		if sequenceNumber == 0 {
+			sequenceNumber = lastSequence + 1
```

Comment:
Would it make sense to set sequence numbers with some gap between them? e.g 10, 20... etc so later a rule can be inserted between without recreating the whole list?

Reply:
From terraform perspective the list is ordered, which means if rule 3.5 is inserted between rule 3 and rule 4, this will result in diff for rule 4, and a new rule 5. As a result, we will update all rules starting from the inserted rule, hence we don't need to bother about gaps in sequence numbers.